web3/main.py

```diff
@@ -206,3 +206,17 @@ def ens(self):
     @ens.setter
     def ens(self, new_ens):
         self._ens = new_ens
+
+    @property
+    def pm(self):
+        if self._pm is not None:
+            return self._pm
+        else:
+            raise AttributeError("The Package Management feature is disabled by default until "
+                "its API stabilizes. To use these features, please enable them by running "
+                "`w3.enable_unstable_package_management_api()` and try again."
+            )
+
+    def enable_unstable_package_management_api(self):
+        from web3.pm import PM
```

Comment:
This was the quick and easy solution to avoiding the circular import this time around, but it seems to me like this won't be suitable long-term when we want to make the PM api available on the `web3` object by default.

Reply:
I'm missing it: which circular import? Does `web3.pm` import from `web3.main` somehow?

I don't follow how this will be different from any other modules that are attached on `Web3` by default.